vmm/src/api/dbus/mod.rs

```diff
@@ -282,30 +282,41 @@ impl DBusApi {
             .await
             .map(|_| ())
     }
+
+    #[dbus_interface(signal)]
+    async fn event(ctxt: &zbus::SignalContext<'_>, event: String) -> zbus::Result<()> {}
```

Comment:
I assume this is intentionally blank, while being able to produce the `event` string to users. Any pointers to explain this behavior?

Reply:
Ah, I should've mentioned this in a comment. The implementation for the `Event` signal is provided by the `dbus_interface` macro based on the function signature, here's the output of `cargo expand` for that function:
```rust
async fn event(
    ctxt: &zbus::SignalContext<'_>,
    event: String,
) -> zbus::Result<()> {
    ctxt.connection()
        .emit_signal(
            ::std::option::Option::None::<()>,
            ctxt.path(),
            <DBusApi as ::zbus::Interface>::name(),
            "Event",
            &(event,),
        )
        .await
}
```